test/conformance/gateway_conformance_test.go

```diff
@@ -62,6 +60,8 @@ var (
 		// cannot support the path > method > header precedence,
 		// but no way to omit individual cases.
 		tests.HTTPRouteMethodMatching.ShortName,
+		// only expression router supports query param matches
+		tests.HTTPRouteQueryParamMatching.ShortName,
```

Comment:
Did you check whether `test/conformance/gateway_expermimental_conformance_test.go` test it too?
```
make test.conformance-experimental
```
the above should test it too

Reply:
Added it.